Sources/Hyperwallet.swift

```diff
@@ -420,15 +420,50 @@ public final class Hyperwallet {
     /// if the current one is expired or is about to expire.
     ///
     /// - Parameters:
-    ///   - pagination: the ordering and filtering criteria
+    ///   - queryParam: the ordering and filtering criteria
     ///   - completion: the callback handler of responses from the Hyperwallet platform
-    public func listTransferMethods(pagination: HyperwalletTransferMethodPagination? = nil,
+    public func listTransferMethods(queryParam: HyperwalletTransferMethodQueryParam? = nil,
                                     completion: @escaping (HyperwalletPageList<HyperwalletTransferMethod>?,
         HyperwalletErrorType?) -> Void) {
         httpTransaction.performRest(httpMethod: .get,
                                     urlPath: "users/%@/transfer-methods",
                                     payload: "",
-                                    pagination: pagination,
+                                    queryParam: queryParam,
+                                    completionHandler: completion)
+    }
+
+    /// Returns the list of receipts for the User associated with the authentication token
+    ///
+    /// The ordering and filtering of `HyperwalletReceipt` will be based on the criteria specified within the
+    /// `HyperwalletBankAccountQueryParam` object, if it is not nil. Otherwise the default ordering and
+    /// filtering will be applied.
+    ///
+    /// * Offset: 0
+    /// * Limit: 10
+    /// * Created Before: N/A
+    /// * Created After: N/A
+    /// * Currency: All
+    /// * Sort By: Created On
+    ///
+    /// The `completion: @escaping (HyperwalletPageList<HyperwalletReceipt>?, HyperwalletErrorType?) -> Void`
+    /// that is passed in to this method invocation will receive the successful
+    /// response(HyperwalletPageList<HyperwalletReceipt>?) or error(HyperwalletErrorType) from processing
+    /// the request.
+    ///
+    /// This function will request a new authentication token via `HyperwalletAuthenticationTokenProvider`
+    /// if the current one is expired or is about to expire.
+    ///
+    /// - Parameters:
+    ///   - queryParam: the ordering and filtering criteria
+    ///   - completion: the callback handler of responses from the Hyperwallet platform
+
+    public func listTransactionReceipts(queryParam: HyperwalletReceiptQueryParam? = nil,
```

Comment:
Can you please change this method name to ```listUserReceipts```?

Reply:
np, fixed